letter-sender/src/main/java/com/christmas/letter/sender/model/Letter.java

```diff
@@ -0,0 +1,15 @@
+package com.christmas.letter.sender.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Letter {
+
+  private String email;
```

Comment:
Let's use annotations here to make sure the user cannot provide an empty email, or name, etc

Reply:
solved wirh:
  @NotNull(message="Email required")
  @Email(regexp = "[a-z0-9._%+-]+@[a-z0-9.-]+\\.[a-z]{2,3}")
  private String email;